README.md

```diff
@@ -81,4 +81,4 @@ Read our contributing [guide](CONTRIBUTING.md) to get started with contributing
 
 Thank you to all the contributors who have helped us in making this project better 🙌
 
-<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a>
+<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a>  <a href="https://github.com/wdmcquinn"><img src="https://github.com/wdmcquinn.png" width="30" /></a>
```

Comment:
`/></a>  <a href`

can you remove space here.
we would not want to show spaces

Reply:
Okay sure